Configuration/StandardSequences/python/RawToDigi_DataMapper_cff.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.RawToDigi_Data_cff import *
+from EventFilter.RawDataCollector.rawDataMapperByLabel_cfi import rawDataMapperByLabel
+
+RawToDigi.insert(0, rawDataMapperByLabel)
+RawToDigi_noTk.insert(0, rawDataMapperByLabel)
+RawToDigi_pixelOnly.insert(0, rawDataMapperByLabel)
```

Comment:
it would be much easier to maintain this via the modifier mechanisms. [generally modifying objects in files other than where they are defined is bad practice]

Reply:
as of now, we do not have a data-specific modifier.
Hence this solution.